apps/codeforafrica/src/components/Page/Page.js

```diff
@@ -3,13 +3,15 @@ import PropTypes from "prop-types";
 import React from "react";
 
 import Footer from "@/codeforafrica/components/Footer";
+import NavBar from "@/codeforafrica/components/NavBar";
 
-function Page({ children, footer, title }) {
+function Page({ children, footer, navbar, title }) {
   return (
     <>
       <Head>
         <title>{title}</title>
       </Head>
+      <NavBar menu={navbar?.menu} />
```

Comment:
```suggestion
      <NavBar {...navbar} />
```
I take it whatever is in `navbar` is meant for `NavBar`.